packages/headless/src/features/configuration/configuration-slice.ts

```diff
@@ -156,3 +171,23 @@ export const configurationReducer = createReducer(
           action.payload.tab || state.analytics.originLevel2;
       })
 );
+
+function handleUpdateBasicConfiguration(
+  state: ConfigurationState,
+  payload: UpdateBasicConfigurationActionCreatorPayload
+) {
+  if (payload.accessToken) {
+    state.accessToken = payload.accessToken;
+  }
+  if (payload.organizationId) {
+    state.organizationId = payload.organizationId;
+  }
+  if (payload.platformUrl) {
+    state.platformUrl = payload.platformUrl;
+    state.search.apiBaseUrl = `${payload.platformUrl}${searchAPIEndpoint}`;
+    state.analytics.apiBaseUrl = legacyAnalyticsUrlFromPlatformUrl(
+      payload.platformUrl,
+      state.organizationId
+    );
```

Comment:
Do we actually need to set these urls when handling `updateCommerceBasicConfiguration`? I don't think they're used for commerce

Reply:
On the other hand, there's no harm in setting those (it's not costly by any means in terms of performance).

Reply:
Yeah, I don't mind leaving this in